coderd/database/migrations/000274_workspace_modules.up.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE
+    workspace_resources
+ADD
+    COLUMN module_path TEXT;
+
+CREATE TABLE workspace_modules (
+    id uuid NOT NULL,
+    job_id uuid NOT NULL REFERENCES provisioner_jobs (id) ON DELETE CASCADE,
```

Comment:
pair review (non-blocking): alternatively we could normalize this and instead have a `workspace_resources.module_id` column reference an individual row of the `workspace_modules` table so we don't end up storing multiple instances of the same module.

Reply:
That's completely fair 👍 Thanks for clarifying!